client/src/views/tx.js

```diff
@@ -30,6 +30,9 @@ export default ({ t, tx, tipHeight, spends, openTx, page, unblinded, ...S }) =>
   <div>
     <div className="transaction-page">
       <div className="container">
+        <a className="back-link" href="tx/recent">
+          <img src="img/icons/left-arrow.png"/><span>{t`All Transactions`}</span>
```

Comment:
This link doesn't seem particularly useful to me (I rarely want to navigate to the recent mempool transactions when already looking at a particular transaction) and is taking prime screen real-estate. I would consider dropping this and probably "All Blocks" too ("All Assets" does seem more potentially useful, though).